### PR TITLE
Display all smallest test cases

### PR DESCRIPTION
**Pull request by scott113341**
In the event of a tie in the size comparison, only one framework will be shown as "smallest":

image

Credit goes to @mmedal for the [discovery](https://github.com/rtsao/styletron/pull/57#issuecomment-269085873).

### CODE REVIEW

File: src/run.js [hunk 1]
```diff
@@ -26,13 +26,15 @@ export const runTest = (testName, cases) => {
...
         testSuite.on('complete', function onComplete() {
             let smallestSize = Number.MAX_VALUE;
-            let smallest = '?';
+            let smallest = [];
             Object.keys(testCases).forEach((caseName) => {
                 const length = testCases[caseName].result.length;
                 console.log(`${caseName} length`, length);
-                if (smallestSize > length) {
+                if (smallestSize === length) {
+                    smallest = [...smallest, caseName];
```
Comment (PhiLhoSoft) [inline src/run.js]: Is `smallest.push(caseName)` too old-school? 😜

Comment (scott113341) [inline src/run.js]: Haha definitely not, changed to be more clear.

File: src/run.js [hunk 2]
```diff
@@ -26,13 +26,15 @@ export const runTest = (testName, cases) => {
 
         testSuite.on('complete', function onComplete() {
             let smallestSize = Number.MAX_VALUE;
-            let smallest = '?';
+            let smallest = [];
             Object.keys(testCases).forEach((caseName) => {
                 const length = testCases[caseName].result.length;
                 console.log(`${caseName} length`, length);
-                if (smallestSize > length) {
+                if (smallestSize === length) {
```
Comment (scott113341) [inline src/run.js]: Yes, that sounds good.  I've updated my PR with that functionality, plus some extra goodies.  GitHub doesn't show colors, but I must admit I'm too lazy to add coloring to the size table for now.  Resulting output:

```text
Running simple test.

  aphrodite               582 B  (291 B gzipped)
  aphrodite-no-important  472 B  (284 B gzipped)
  cssobj                  631 B  (323 B gzipped)
  cxs                     552 B  (304 B gzipped)
  cxs-optimized           634 B  (327 B gzipped)
  fela                    500 B  (292 B gzipped)
  free-style              521 B  (296 B gzipped)
  glamor                  591 B  (314 B gzipped)
  j2c                     748 B  (337 B gzipped)
  jss                     636 B  (333 B gzipped)
  jss-without-preset      636 B  (344 B gzipped)
  styletron               502 B  (293 B gzipped)
  
  Smallest is:          aphrodite-no-important
  Smallest gzipped are: aphrodite-no-important, aphrodite, fela, styletron, free-style


  aphrodite              x  6,364 ops/sec ±12.27% (6 runs sampled)
  aphrodite-no-important x  6,723 ops/sec ±4.43% (6 runs sampled)
  cssobj                 x  3,310 ops/sec ±7.32% (6 runs sampled)
  cxs                    x  8,465 ops/sec ±8.68% (6 runs sampled)
  cxs-optimized          x  6,492 ops/sec ±6.53% (6 runs sampled)
  fela                   x 45,262 ops/sec ±9.77% (5 runs sampled)
  free-style             x 13,732 ops/sec ±8.93% (6 runs sampled)
  glamor                 x  4,305 ops/sec ±7.07% (6 runs sampled)
  j2c                    x 19,760 ops/sec ±7.61% (6 runs sampled)
  jss                    x 14,769 ops/sec ±7.68% (5 runs sampled)
  jss-without-preset     x 23,519 ops/sec ±7.83% (6 runs sampled)
  styletron              x 88,361 ops/sec ±4.71% (6 runs sampled)

  Fastest is: styletron
  

Running nested test.

  aphrodite               1,179 B  (415 B gzipped)
  aphrodite-no-important    926 B  (399 B gzipped)
  cssobj                  1,100 B  (422 B gzipped)
  cxs                       975 B  (394 B gzipped)
  cxs-optimized           1,061 B  (419 B gzipped)
  fela                      800 B  (404 B gzipped)
  free-style                802 B  (370 B gzipped)
  glamor                  1,479 B  (452 B gzipped)
  j2c                     1,350 B  (427 B gzipped)
  jss                     1,089 B  (429 B gzipped)
  jss-without-preset      1,089 B  (439 B gzipped)
  styletron                 790 B  (409 B gzipped)
  
  Smallest are:         styletron, fela, free-style
  Smallest gzipped is:  free-style


  aphrodite              x  2,749 ops/sec ±4.45% (6 runs sampled)
  aphrodite-no-important x  2,948 ops/sec ±10.82% (5 runs sampled)
  cssobj                 x  1,578 ops/sec ±9.77% (6 runs sampled)
  cxs                    x  4,292 ops/sec ±8.50% (6 runs sampled)
  cxs-optimized          x  3,782 ops/sec ±3.14% (6 runs sampled)
  fela                   x 19,406 ops/sec ±2.76% (6 runs sampled)
  free-style             x  5,445 ops/sec ±7.89% (5 runs sampled)
  glamor                 x  1,918 ops/sec ±5.76% (6 runs sampled)
  j2c                    x 10,883 ops/sec ±11.40% (6 runs sampled)
  jss                    x  3,571 ops/sec ±15.13% (6 runs sampled)
  jss-without-preset     x  5,144 ops/sec ±4.25% (6 runs sampled)
  styletron              x 29,352 ops/sec ±6.12% (6 runs sampled)

  Fastest is: styletron
  

Running style overload test.

  aphrodite               3,537 B  (605 B gzipped)
  aphrodite-no-important  2,866 B  (592 B gzipped)
  cssobj                  3,165 B  (464 B gzipped)
  cxs                     2,757 B  (564 B gzipped)
  cxs-optimized           2,874 B  (579 B gzipped)
  fela                    1,528 B  (397 B gzipped)
  free-style              2,491 B  (525 B gzipped)
  glamor                  3,034 B  (607 B gzipped)
  j2c                     4,320 B  (497 B gzipped)
  jss                     3,318 B  (626 B gzipped)
  jss-without-preset      3,318 B  (639 B gzipped)
  styletron               1,549 B  (402 B gzipped)
  
  Smallest are:         fela, styletron
  Smallest gzipped are: fela, styletron


  aphrodite              x    917 ops/sec ±9.62% (5 runs sampled)
  aphrodite-no-important x  1,056 ops/sec ±6.79% (6 runs sampled)
  cssobj                 x    693 ops/sec ±7.70% (6 runs sampled)
  cxs                    x  1,492 ops/sec ±3.16% (6 runs sampled)
  cxs-optimized          x  1,156 ops/sec ±11.54% (6 runs sampled)
  fela                   x 14,802 ops/sec ±2.77% (6 runs sampled)
  free-style             x  2,171 ops/sec ±8.50% (6 runs sampled)
  glamor                 x    790 ops/sec ±7.05% (6 runs sampled)
  j2c                    x  3,617 ops/sec ±7.87% (5 runs sampled)
  jss                    x  3,279 ops/sec ±4.03% (6 runs sampled)
  jss-without-preset     x  4,081 ops/sec ±18.64% (6 runs sampled)
  styletron              x 21,953 ops/sec ±7.06% (5 runs sampled)

  Fastest is: styletron
  

Running classes overload test.

  aphrodite               2,955 B  (406 B gzipped)
  aphrodite-no-important  2,504 B  (399 B gzipped)
  cssobj                  2,795 B  (393 B gzipped)
  cxs                     1,334 B  (283 B gzipped)
  cxs-optimized           1,713 B  (297 B gzipped)
  fela                    1,078 B  (259 B gzipped)
  free-style              1,173 B  (270 B gzipped)
  glamor                  1,283 B  (291 B gzipped)
  j2c                     3,950 B  (420 B gzipped)
  jss                     2,896 B  (402 B gzipped)
  jss-without-preset      2,896 B  (414 B gzipped)
  styletron               1,099 B  (260 B gzipped)
  
  Smallest are:         fela, styletron
  Smallest gzipped are: fela, styletron, free-style


  aphrodite              x  1,124 ops/sec ±8.04% (6 runs sampled)
  aphrodite-no-important x  1,237 ops/sec ±5.08% (6 runs sampled)
  cssobj                 x    769 ops/sec ±11.80% (5 runs sampled)
  cxs                    x  1,950 ops/sec ±5.71% (6 runs sampled)
  cxs-optimized          x  1,473 ops/sec ±3.62% (5 runs sampled)
  fela                   x 31,421 ops/sec ±5.74% (6 runs sampled)
  free-style             x  3,193 ops/sec ±6.66% (6 runs sampled)
  glamor                 x  3,361 ops/sec ±6.40% (6 runs sampled)
  j2c                    x  4,110 ops/sec ±6.19% (6 runs sampled)
  jss                    x  3,280 ops/sec ±42.81% (5 runs sampled)
  jss-without-preset     x  5,693 ops/sec ±3.59% (6 runs sampled)
  styletron              x 45,929 ops/sec ±4.43% (6 runs sampled)

  Fastest is: styletron
```

**Comment by PepijnSenders** [comment]
Thanks for contributing @scott113341, LGTM :)

**Pull request merged**